remoter-server/README.md

```diff
@@ -14,4 +14,4 @@ with the argument being the JSON-ified string of the task that is to be executed
 
 For example:
 
```

Comment:
in the readme for the text saying `Make sure you have the server running locally.` can you specify in the readme how to do it?

Reply:
Done